### PR TITLE
NO-JIRA: feat: added aliases for edge-enablement

### PR DESCRIPTION
**Pull request by jeff-roche**
Adding in the reviewers and approvers lists for the edge enablement team

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -390,3 +390,17 @@ aliases:
   - pmtk
   - eggfoobar
   - pliurh
+edge-enablement-reviewers:
```
Comment (suleymanakbas91) [inline OWNERS_ALIASES:393]: Indentation seems to be off: 

Screenshot 2024-01-29 at 09 31 14

**Review by suleymanakbas91 — CHANGES_REQUESTED** [review]

File: OWNERS_ALIASES (hunk 1 above)
Comment (suleymanakbas91) [reply OWNERS_ALIASES:393]: Is there a reason why Bulat is not in the reviewers?

Comment (jeff-roche) [reply OWNERS_ALIASES]: No reason other than I just missed it. I fixed the spacing and added Bulat to reviewers

**Comment by jeff-roche** [comment]
@danilo-gemoli how do I retrigger the tests?

**Pull request merged**